src/client/chunk_closure.cpp

```diff
@@ -366,6 +375,28 @@ void ClientClosure::OnEpochTooOld() {
         << butil::endpoint2str(cntl_->remote_side()).c_str();
 }
 
+void ClientClosure::OnGetAuthTokenFail() {
+    reqDone_->SetFailed(status_);
+    LOG(ERROR) << OpTypeToString(reqCtx_->optype_)
+        << " get auth token failed, reqCtx: " << *reqCtx_
+        << ", retried times: " << reqDone_->GetRetriedTimes()
+        << ", IO id: " << reqDone_->GetIOTracker()->GetID()
+        << ", request id: " << reqCtx_->id_
+        << ", remote side: "
+        << butil::endpoint2str(cntl_->remote_side()).c_str();
+}
+
+void ClientClosure::OnAuthFail() {
+    reqDone_->SetFailed(status_);
+    LOG(ERROR) << OpTypeToString(reqCtx_->optype_)
```

Comment:
How about this? What happens if last ticket is expired and we cann't refresh new ticket from mds, will this cause IO error?